.travis.yml

```diff
@@ -3,4 +3,8 @@ language: go
 go:
```

Comment:
No longer need to have it as golang since that will run inside container.

Change it to generic.. like here 

https://github.com/ahab94/ApnaSchool/blob/dev/.travis.yml